services/ui-src/src/components/tables/SortableDrawerReportPageTable.tsx

```diff
@@ -0,0 +1,153 @@
+import { Button, Image } from "@chakra-ui/react";
```

Comment:
non-blocking: since this file is strictly for standards right now, would it make sense to call it something like `SortableNaaarStandardsTable` or `SortableStandardsTable` or something? I know we might allow more entities in the future, but just a thought since the page type isn't what makes this one unique